types/order.d.ts

```diff
@@ -0,0 +1,78 @@
+export interface Options {
+    readonly fields?: Array<string>,

```

Comment:
We've a mix of `Array<string>` and `string[]` :(. I know that in Registry it's using this form but let's please go with `[]` since it's the most common Typescript idiom.

Reply:
Depois porto isto para o ripe-registry-api-js